CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+### 1.0.0
+* added shared stylelint config to configure a couple of rules to fit with styled-components’ style of writing CSS
```

Comment:
Wasn't this added in https://github.com/styled-components/stylelint-processor-styled-components/commit/cf988ad492586def7ca2454a161ad216aa280129, which was released with 0.4.0?

Reply:
Good point, moved it!